jupyter_client/multikernelmanager.py

```diff
@@ -104,7 +104,6 @@ def start_kernel(self, kernel_name=None, **kwargs):
         km = self.kernel_manager_factory(connection_file=os.path.join(
                     self.connection_dir, "kernel-%s.json" % kernel_id),
                     parent=self, log=self.log, kernel_name=kernel_name,
-                    **kwargs
```

Comment:
This line should be left alone, since `start_kernel` is meant as a passthrough for the single kernel-manager constructor.

If nothing else, one line above kernel_spec_manager is added to kwargs.